packages/catalog-search/src/SearchFilters.jsx

```diff
@@ -1,22 +1,19 @@
 import React, { useMemo, useContext } from 'react';
 import PropTypes from 'prop-types';
 import { breakpoints } from '@edx/paragon';
+import { useWindowSize } from '@edx/frontend-enterprise-utils';
```

Comment:
This hook is now in Paragon and is better tested and maintained there, should we take this opportunity to switch it out?

Reply:
I had a similar thought. If we are going to remove it, it probably makes sense to do now prior to first publish since it'll become a breaking change if we remove it once `@edx/frontend-enterprise-utils` is already published to NPM.

I will update!